### PR TITLE
🐛 Only bind the agent role for the addon group

### PR DESCRIPTION
**Pull request by zhujian7**
## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/addon/controllers/addontemplate/controller.go [hunk 1]
```diff
@@ -210,8 +219,19 @@ func (c *addonTemplateController) runController(
 	if err != nil {
 		return err
 	}
-
 	kubeInformers.Start(ctx.Done())
 
+	// trigger the manager to reconcile for the existing managed cluster addons
+	mcas, err := c.addonInformers.Addon().V1alpha1().ManagedClusterAddOns().Lister().List(labels.Everything())
+	if err != nil {
+		return err
```
[the remark this answers is not in the document]
Comment (zhujian7) [reply pkg/addon/controllers/addontemplate/controller.go]: @qiujian16 PTAL

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**